frontend/package.json

```diff
@@ -14,8 +14,8 @@
     "lint": "stylelint '**/*.scss' && prettier --check './src' && next lint",
     "licensecheck": "license-checker --excludePrivatePackages --production --onlyAllow 'MPL; MIT; BSD-3-Clause; BSD-2-Clause; 0BSD; Apache-2.0; ISC; Zlib; CC0; Unlicense; WTFPL; Unicode-DFS-2016;'",
     "test": "jest",
-    "postinstall": "msw init public/ --save",
-    "prepare": "cd .. && husky install frontend/.husky"
+    "postinstall": "if type msw > /dev/null; then msw init public/ --save; else echo 'msw not installed'; fi",
```

Comment:
Hmm, I don't understand why this would be necessary, as msw is just listed as a dependency :/ Yes, it's just a devDependency, but the `NODE_ENV=\"development\"` in the root `package.json`'s `heroku-postbuild` script [should take care of that](https://devcenter.heroku.com/articles/nodejs-support#skip-pruning). And I especially don't understand why this would be necessary _now_, i.e. how would the removal of Gulp affect this?

But msw shouldn't be necessary on dev anyway, so if this makes it work, that's good enough I guess :shrug: